web/src/js/__tests__/components/EventLogSpec.js

```diff
@@ -0,0 +1,57 @@
+jest.mock('../../components/EventLog/EventList')
+
+import React from 'react'
+import renderer from 'react-test-renderer'
+import TestUtils from 'react-dom/test-utils'
+import ConnectedComponent, { EventLog } from '../../components/EventLog'
+import { Provider } from 'react-redux'
+import { TStore } from '../ducks/tutils'
+
+window.addEventListener = jest.fn()
+window.removeEventListener = jest.fn()
+
+describe('EventLog Component', () => {
+    let store = TStore(),
+        provider = renderer.create(
+        <Provider store={store}>
+            <ConnectedComponent/>
+        </Provider>),
+        tree = provider.toJSON()
+
+    it('should connect to state and render correctly', () => {
+        expect(tree).toMatchSnapshot()
+    })
+
+    it('should handl toggleFilter', () => {
+        let debugToggleButton = tree.children[0].children[1].children[0]
```

Comment:
Yes, that looks weird, but that's the only way I came up with to cover `() => toogleFilter(type)`.